habitat-lab/habitat/config/default.py

```diff
@@ -8,17 +8,25 @@
 import os.path as osp
 import threading
 from functools import partial
-from typing import List, Optional
+from typing import TYPE_CHECKING, List, Optional
 
 from hydra import compose, initialize_config_dir
-from omegaconf import DictConfig, OmegaConf
```

Comment:
Looks like this is being imported from here elsewhere in the code causing some Test failures.